kintaro/kintaro.py

```diff
@@ -115,6 +115,16 @@ def _parse_field(self, key, value, field_data):
             key = '${}'.format(key)
         if field_data['translatable']:
             key = '{}@'.format(key)
+
+        # Handle ReferenceField as doc reference.
+        if field_data['type'] == 'ReferenceField' and isinstance(value, dict):
+            for binding in self.config.bind:
+                if binding.kintaro_collection == value['collection_id']:
+                    content_path = '{}{}.yaml'.format(
+                        binding.collection, value['document_id'])
```

Comment:
Need to ensure this is separated by a `/`. Currently `binding.collection` has an optional trailing slash.